liquibase-core/src/main/java/liquibase/database/Database.java

```diff
@@ -472,5 +472,7 @@ default String getDisplayName() {
     String unescapeDataTypeString(String dataTypeString);
 
     ValidationErrors validate();
+
+    boolean supportsCreateIfNotExists(Class<? extends DatabaseObject> type);
```

Comment:
Great, this way we can validate if it supports any kind of "exists" syntax. Could you please add simple javadocs explaining this method?